src/olympia/devhub/templates/devhub/email/submission.txt

```diff
@@ -1,23 +1,23 @@
-Subject: Mozilla Add-ons: {{ add-on name }} has been submitted to addons.mozilla.org! 
+Subject: Mozilla Add-ons: {{ addon_name }} has been submitted to addons.mozilla.org!
 
-Thanks for submitting your {{ add-on name }} add-on to addons.mozilla.org (AMO)! Your add-on listing will go live shortly in our gallery here: {{ url }}
+Thanks for submitting your {{ addon_name }} add-on to addons.mozilla.org (AMO)! Your add-on listing will go live shortly in our gallery here: {{ detail_url }}
 
 Please keep in mind that while your add-on has been screened and approved for listing, other reviewers may look into it in the future and determine that it requires changes or should be removed from the gallery. If that occurs, you will receive a separate notification with details and next steps.
 
-For more information about the review process and policies, please visit https://developer.mozilla.org/Add-ons/AMO/Policy/Reviews. 
+For more information about the review process and policies, please visit https://developer.mozilla.org/Add-ons/AMO/Policy/Reviews.
```

Comment:
I think we should avoid `.` at the end of urls too (either drop the `.` or always have a space, e.g. `Reviews .`) but that's more about accounting for differences in how email clients may treat the URL).